PROPOSALS.md

```diff
@@ -120,7 +120,9 @@ The following is a step by step guide for common proposals:
 If your proposal is a "multi-phase proposal" with multiple implementation and deprecation steps, make sure to
 precisely define when each step is supposed to take place.
 
-### A note on stale proposals
+## Notes on the proposal process
```

Comment:
(I'd probably just remove this subheader entirely, going straight ahead with `### Stale proposals`)